packages/wonder-blocks-dropdown/components/action-menu.test.js

```diff
@@ -70,7 +69,7 @@ describe("ActionMenu", () => {
             </ActionMenu>,
         );
 
-        menu.setState({open: true});
+        menu.find(ClickableBehavior).simulate("click");
```

Comment:
This could also be `menu.setProps("opened", true);` I think.  This is fine too.

Reply:
I feel like simulating the click tests it a little more holistically than setting the prop/state